components/tabs/src/fixtures.js

```diff
@@ -7,8 +7,7 @@ import Table from '@govuk-react/table';
 
 import Tabs from '.';
 
-const flip2dArray = (prev, next) =>
-  next.map((item, index) => [...(prev[index] || []), next[index]]);
+const flip2dArray = (prev, next) => next.map((item, index) => [...(prev[index] || []), next[index]]);
```

Comment:
now that must be more than 120 characters

Reply:
nope - it's less :grin: